From 1c10956f0283edcee4b5004d406eecee1fcb7c18 Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Mon, 11 Jun 2018 17:50:17 +0530
Subject: [PATCH 0362/1852] drm: xlnx: vtc: Fix the kernel-doc warnings

commit 892b010a757acde8788776ab3354b9b7f18d1b20 from
https://github.com/Xilinx/linux-xlnx.git

Fix the kernel-doc warnings by fixing the initial short description
of structures.

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/xlnx/xlnx_vtc.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/gpu/drm/xlnx/xlnx_vtc.c b/drivers/gpu/drm/xlnx/xlnx_vtc.c
index dc3ba2c7209e..4776bb290f0c 100644
--- a/drivers/gpu/drm/xlnx/xlnx_vtc.c
+++ b/drivers/gpu/drm/xlnx/xlnx_vtc.c
@@ -163,7 +163,7 @@
 				VTC_IXR_D_ALL_MASK |		\
 				VTC_IXR_LOCKALL_MASK)
 /**
- * struct xilinx_vtc: Xilinx VTC object
+ * struct xilinx_vtc - Xilinx VTC object
  *
  * @base: base addr
  * @irq: irq
@@ -178,7 +178,7 @@ struct xilinx_vtc {
 };
 
 /**
- * struct xilinx_vtc_polarity: vtc polarity config
+ * struct xilinx_vtc_polarity - vtc polarity config
  *
  * @active_chroma: active chroma polarity
  * @active_video: active video polarity
@@ -199,7 +199,7 @@ struct xilinx_vtc_polarity {
 };
 
 /**
- * struct xilinx_vtc_hori_offset: vtc horizontal offset config
+ * struct xilinx_vtc_hori_offset - vtc horizontal offset config
  *
  * @vblank_hori_start: vblank horizontal start
  * @vblank_hori_end: vblank horizontal end
@@ -214,7 +214,7 @@ struct xilinx_vtc_hori_offset {
 };
 
 /**
- * struct xilinx_vtc_src_config: vtc source config
+ * struct xilinx_vtc_src_config - vtc source config
  *
  * @field_id_pol: filed id polarity source
  * @active_chroma_pol: active chroma polarity source
-- 
2.31.1

